artifacts/Dockerfile

```diff
@@ -72,6 +72,9 @@ ENV icu4j_jar_path=${base_path}/libs-release-local/icu4j
 # environment variable for i18n zip path
 ENV i18n_zip_path=${base_path}/libs-release-local/i18n
 
+# environment variable for idp auth wrapper zip path
+ENV idp_auth_wrapper_lib_zip_path=${base_path}/libs-release-local/idp/idp-auth-wrapper
```

Comment:
change the variable "idp_auth_wrapper_lib_zip_path" to "idp_auth_wrapper_jar_path"

Reply:
Add below section under RUN wget command

```
wget ${auth_wrapper_url_env} -O ${idp_auth_wrapper_jar_path}/authentication-wrapper.jar &&\
echo "Downloaded IDP auth wrapper jar" &&\
```